bescort.lic

```diff
@@ -1474,6 +1489,30 @@ class Bescort
     end
   end
 
+  def ferry_textbook
+    start_script('first-aid')
+    waitfor 'reaches the dock and its crew ties the ferry off'
+    stop_script('first-aid') if Script.running?('first-aid')
+    waitrt?
+    move 'go dock'
+  end
+
+  def ferry_compendium
```

Comment:
What do you think about using a single function that takes the script name as an argument?